kratos/processes/calculate_discontinuous_distance_to_skin_process.cpp

```diff
@@ -76,9 +106,16 @@ namespace Kratos
 		const int number_of_elements = (mFindIntersectedObjectsProcess.GetModelPart1()).NumberOfElements();
 		auto& r_elements = (mFindIntersectedObjectsProcess.GetModelPart1()).ElementsArray();
 
-		#pragma omp parallel for schedule(dynamic)
-		for (int i = 0; i < number_of_elements; ++i) {
-			CalculateElementalDistances(*(r_elements[i]), rIntersectedObjects[i]);
+		if (mOptions.Is(CALCULATE_ELEMENTAL_EDGE_DISTANCES)) {
+			#pragma omp parallel for schedule(dynamic)
```

Comment:
@RiccardoRossi @philbucher do you know if it is possible to set the `schedule(dynamic)` clause with the `ParallelUtils`?

Reply:
At the moment not
Why do you need this?

Reply:
Because the number of intersected objects can vary hugely among elements.

Reply:
I see
This does sound like a good usecase for dynamic scheduling

in principle it would be easy to add/expose this functionality (when using OpenMP)
However it would be difficult/more work to implement it with C++11 based parallelism as there as of now the partitioning is done at the beginning. => we would need to implement our own repartition for this to work.

having said this I am not opposed to adding it to the interface as it makes sense to me.

then we can think later how to implement it nicely. (In the worst case it would not be dynamic right now but would still work)

@rubenzorrilla can you make a note abt this for the next KTC meeting?

Reply:
Sure!